backend/templates/proxy_host.conf

```diff
@@ -4,6 +4,8 @@
 
 {% include "_hsts_map.conf" %}
 
+{% include "_resolved_proto_map.conf" %}
```

Comment:
Since this conf you're including doesn't change content per host, you can instead put the here:
```
/docker/rootfs/etc/nginx/conf.d/include/resolved_progo_map.conf
```
and then include it using nginx include:
```
include conf.d/include/resolved_progo_map.conf;
```

Reply:
Thanks, changes were commited.